stixcore/io/fits/processors.py

```diff
@@ -91,8 +91,12 @@ def generate_filename(cls, product, *, version, date_range, status=''):
         if 'tc_packet_seq_control' in product.control.colnames and user_req != '':
             tc_control = f'-{product.control["tc_packet_seq_control"][0]:05d}'
 
+        incomplete = ''
+        if not header and product.level != 'LB' and product.fits_daily_file is True:
+            incomplete = 'U'
+
         return f'solo_{product.level}_stix-{product.type}-{product.name.replace("_", "-")}' \
-               f'_{date_range}_V{version:02d}{status}{user_req}{tc_control}.fits'
+               f'_{date_range}_V{version:02d}{incomplete}{status}{user_req}{tc_control}.fits'
```

Comment:
This was the idea of the status flag so status could be `C`, `I` or `U` why add another variable?

Reply:
wow did not realized that this was already foreseen a time ago
will use the status